test/reactionsystem_core/events.jl

```diff
@@ -424,21 +424,22 @@ let
     osol_events = solve(oprob_events, Tsit5())
     @test osol == osol_events
 
-    # Checks for SDE simulations.
+    # Checks for SDE simulations (note, non-seed dependant test should be created instead).
```

Comment:
Basically checking that callbacks and events produce the same result. One alternative is to simply drop them (we already check that all event types work, and also check that callbacks and events are the same for ODEs).